app-accessibility/emacspeak-ss/emacspeak-ss-1.9.1-r1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Adds support for several speech synthesizers to emacspeak"
+HOMEPAGE="http://leb.net/blinux/"
+SRC_URI="http://leb.net/pub/blinux/emacspeak/blinux/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND=">=app-accessibility/emacspeak-18"
+
+PATCHES=(
+	"${FILESDIR}"/gentoo-apollo-fix.patch
+)
+
+src_prepare() {
+	default
+	tc-export CC
+}
+
+src_install() {
+	emake \
+		prefix="${D}"/usr \
```

Comment:
${ED} for both

Reply:
Is for the EPREFIX thing, right?

What is that?, where can I find info about it?

Reply:
Yes - exactly.

Take a look at https://wiki.gentoo.org/wiki/Project:Prefix (maybe do a quick install of one). It's very cool.